staging/src/k8s.io/client-go/tools/clientcmd/api/helpers.go

```diff
@@ -93,13 +95,10 @@ func ShortenConfig(config *Config) {
 	// http://golang.org/pkg/encoding/json/#Marshal using base64 to encode []byte
 	for key, authInfo := range config.AuthInfos {
 		if len(authInfo.ClientKeyData) > 0 {
-			authInfo.ClientKeyData = redactedBytes
+			authInfo.ClientKeyData = dataOmittedBytes
```

Comment:
nit: for the simplicity of PR, maybe it would be better not renaming this variables and do this another PR?

Reply:
In the GitHub viewer it's hard to see because it removes pretty much all the context, but this line change is done in the `ShortenConfig()` function, which via this PR no longer handles the redaction of secrets as that has been offloaded to the new `redactSecrets()` function, so here for clarity I've updated the variable to be the `dataOmittedBytes`.